runtime/compiler/trj9/ilgen/J9IlGeneratorMethodDetails.cpp

```diff
@@ -35,47 +35,35 @@
 #include "ilgen/J9ByteCodeIlGenerator.hpp"
 #include "trj9/env/VMJ9.h"
 
-namespace TR
-{
-
-IlGeneratorMethodDetails & IlGeneratorMethodDetails::operator=(const TR::IlGeneratorMethodDetails & other)
-   {
-   return self()->J9::IlGeneratorMethodDetailsConnector::operator=(other);
-   }
-
-}
-
```

Comment:
...and you'll need to implement create() here to call J9::IlGeneratorMethodDetails::create()

Reply:
That's exactly what I did, but I hit a strange compilation error in trj9/env/j9method.cpp when using the other create function:
`TR::IlGeneratorMethodDetails & details = TR::IlGeneratorMethodDetails::create(storage, this);`
It's like the compiler cannot distinguish between
`inline static TR::IlGeneratorMethodDetails & create(TR::IlGeneratorMethodDetails & target, TR_ResolvedMethod *method);`
and
` static IlGeneratorMethodDetails *create(TR::IlGeneratorMethodDetails &storage, const TR::IlGeneratorMethodDetails & other);`
I thought that `TR_ResolvedMethod *method` was considered different than `TR::IlGeneratorMethodDetails & other`
Changing the name of the second create function to something else solves the problem.

Reply:
I think I am still going to change the name of function I added to 'clone' because it better reflects what it does: it takes an object and it allocates an identical object at the specified address.